.gitignore

```diff
@@ -66,3 +66,4 @@ captures/
 !/.idea/encodings.xml
 !/.idea/copyright/
 !/.idea/compiler.xml
+
```

Comment:
Remove this line. This PR is already difficult to review so remove any unwanted changes in other files.

Reply:
3967178 Fixed.